functions.php

```diff
@@ -625,4 +625,21 @@ function ($block_content, $block): string {
     10,
     2
 );
-// phpcs:enable Generic.Files.LineLength.MaxExceeded
+
+// Unset the Resistance Hub Campaign page when action_options is disabled
+// Remove it from selector within the editor
+add_filter('theme_page_templates', function ($post_templates) {
```

Comment:
Do we need that? Maybe an NRO wants to use the custom design template but not those Actions features